src/bindings/python/src/pyopenvino/graph/partial_shape.cpp

```diff
@@ -34,6 +34,20 @@ void regclass_graph_PartialShape(py::module m) {
 
     shape.def_static("dynamic", &ov::PartialShape::dynamic, py::arg("rank") = ov::Dimension());
 
+    shape.def_static(
+        "dynamic",
+        [](int64_t rank) {
+            return ov::PartialShape::dynamic(ov::Dimension(rank));
```

Comment:
do we need to bypass the same for C++ API?

Reply:
I think it can be a separate story and PR regarding C++.

Reply:
APIs should be aligned :)